pipelines/utilities/junit_to_qase.py

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/python
+import os
+import sys
+from urllib.parse import urlencode
+from datetime import date
+import requests
+from bs4 import BeautifulSoup
+
+
+def collect_test_results(test_cases):
+
+    test_results = []
+
+    for test_case in test_cases:
+
+        test_result = {
+            "case": "",
+            "case_id": "",
+            "suite": "",
+            "suite_id": "",
+            "skipped": "",
+            "failure": ""
+        }
+
+        test_suite = test_case.get("classname")
+        test_result['suite'] = test_suite.split(".")[-1]
```

Comment:
Let's trim the test case name and eliminate anything after `[`. Will this solve the problem of adding duplicated test cases?

Reply:
Updated!